client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/ExecutedSqlCommand.java

```diff
@@ -0,0 +1,62 @@
+package org.evomaster.client.java.instrumentation;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Info related to SQL command execution.
+ *
+ * This class is IMMUTABLE
+ */
+public class ExecutedSqlCommand implements Serializable {
+
+
+    public static transient long FAILURE_EXECUTION_TIME = -1L;
```

Comment:
what is this for? a mutable static variable is dangerous... if really needed, should be under `org.evomaster.client.java.instrumentation.staticstate`

Reply:
or was it supposed to be `final` and not `transient`???

Reply:
either way, should add some JavaDoc for it

Reply:
I think it was supposed to be final instead of transient. I will add some javadoc for it.